assets/sass/style-dark-mode.scss

```diff
@@ -12,7 +12,7 @@
 		--button--color-background-active: var(--global--color-background);
 		--global--color-border: #9ea1a7;
 
-		.site a:focus,
+		.site a:focus:not(.wp-block-button__link):not(.wp-block-file__button),
 		.site a:focus .meta-nav {
```

Comment:
```suggestion
		.site a:focus:not(.wp-block-button__link):not(.wp-block-file__button) .meta-nav {
```

Reply:
This feels very hacky but this is the best I can come up with because of stress.